tests/ballerina-test/src/test/resources/test-src/workers/fork-join-all-specific.bal

```diff
@@ -1,4 +1,4 @@
-function testForkJoinAllOfSpecific()(string[]) {
+function testForkJoinAllOfSpecific() returns (string[]) {
```

Comment:
Shouldn't it be "returns string[]", rather than "returns (string[])". Just thinking, I guess second one will make it a tuple type. Please check and see.

Reply:
According to the grammar it won't be a tuple type, and all the other folks are using a parenthesis to group returns. But I also prefer without the brackets since it will not confuse with tuple types.